info.md

```diff
@@ -6,6 +6,9 @@
 * <i class="fas fa-shield-alt" style="font-size: 1.2em; color:#233e81;"></i><span style="font-size:1.0em;padding-left:12px;">Defender</span>
 * <i class="fas fa-hammer" style="font-size: 1.2em; color:#233e81;"></i><span style="font-size:1.0em;padding-left:12px;">Breaker</span>
 
+### GitHub
+* [Contribute Here](https://github.com/northdpole/www-project-integration-standards/tree/master/scripts/lambda)
```

Comment:
The other link is for Spyros, and not the main repository. The lambda functions are still in the work to be properly released.
```suggestion
* [Contribute Here](https://github.com/OWASP/www-project-integration-standards)
```